tutorials/20_REST_API.ipynb

```diff
@@ -0,0 +1,485 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Haystack with REST API\n",
+    "\n",
+    "- **Level**: Intermediate\n",
```

Comment:
To be honest, I think this might actually be an advanced topic. The user has to work with Docker, requests to REST API, pipeline ymls. They also have to make changes to lots of files which they probably don't fully understand the significance of.